pytripgui/messages.py

```diff
@@ -0,0 +1,37 @@
+from pytripgui import __version__ as pytripgui_version
+from pytrip import __version__ as pytrip_version
+
+about_txt_en = ""
+about_txt_en += "PyTRipGUI Version: " + pytripgui_version + "\n"
+about_txt_en += "PyTRiP Version:" + pytrip_version + "\n"
+about_txt_en += "\n"
+about_txt_en += "(c) 2012 - 2018 PyTRiP98 Developers\n"
+about_txt_en += "    Niels Bassler\n"
+about_txt_en += "    Leszek Grzanka\n"
```

Comment:
@ljelen add your name here, and bump to 2019. :)

Reply:
ok